src/Robo/Tasks/RunTestsTask.php

```diff
@@ -134,6 +134,16 @@ public function dieOnFail() {
     return $this;
   }
 
+  /**
+   * @param string $directory
+   *
+   * @return $this
+   */
+  public function directory($directory) {
```

Comment:
nit: the options in this file are arranged by `@param` type — maybe move this to where he other strings live.